main.py

```diff
@@ -0,0 +1,5 @@
+string = input('enter your string:')
+if(string == string[::-1]):
+  print('String is Palindome')
+else:
+  print('String is not Palindome')
```

Comment:
Great start on this Mike! I like the efficient way of reversing the string. Time to start looking at how you wanna put this into a function to make it reusable. Good start though!